### PR TITLE
Fix, improve & update bitnami tutos

### PR DESCRIPTION
**Pull request by scraly**

### CODE REVIEW

File: pages/platform/kubernetes-k8s/backing-up-volumes-using-stash/guide.en-asia.md [hunk 1]
```diff
@@ -171,6 +171,13 @@ secret/s3-secret created
 
 The easiest way to install Stash is via Helm, using the chart from [AppsCode Charts Repository](https://github.com/appscode/charts).
 
+First of all, you need to [get a license](https://stash.run/docs/v2021.11.24/setup/install/community/#get-a-license).
+
+TODO: Note if you want to use an enterprise edition please folwo this link instead...
```
Comment (OVH-Team-Guides) [inline pages/platform/kubernetes-k8s/backing-up-volumes-using-stash/guide.en-asia.md]: Hello @scraly, shouldn't this line be removed ? Also "folwo" needs to be changed into "follow"

File: pages/platform/kubernetes-k8s/installing-helm/guide.en-asia.md [hunk 2]
```diff
@@ -66,6 +66,17 @@ The simplest way to install Helm is grabbing the binary release for your platfor
 > [!primary]
 > To use alternative installation procedures, like package managers (Homebrew, Snap etc.), please refer to the [official installation doc](https://docs.helm.sh/using_helm/#installing-helm){.external}.
 
+To check if the `helm` CLI is correctly installed locally, you can display its version:
+
+```bash
+helm version
+```
+
+You can show the installed version on my side:
```
Comment (OVH-Team-Guides) [inline pages/platform/kubernetes-k8s/installing-helm/guide.en-asia.md]: "on my side" -> "on your side" ?

File: pages/platform/kubernetes-k8s/vrack-example-k8s-and-pci/guide.en-gb.md [hunk 3]
```diff
@@ -94,22 +94,39 @@ In the fourth step of creation, we attach it to the private network we created b
 
 ![Setting-up PCI - Attaching to a private network](images/vrack-example-05.png){.thumbnail}
 
-After instance creation, we can see the connection details in the OVHcloud Control Panel. If we log in to the instance using SSH, we can see that it has two network interfaces, one attached to the public IP address we use to log in, the other attached to the private network:
+After instance creation, click on the name of the new created instance, and then scroll down a little in order to see the connection details in the OVHcloud Control Panel.
```
Comment (OVH-Team-Guides) [inline pages/platform/kubernetes-k8s/vrack-example-k8s-and-pci/guide.en-gb.md]: "click on the name of the new created instance" -> "click on the name of the newly created instance"

File: pages/platform/kubernetes-k8s/vrack-example-k8s-and-pci/guide.en-gb.md [hunk 4]
```diff
@@ -94,22 +94,39 @@ In the fourth step of creation, we attach it to the private network we created b
...
 
-<pre class="console"><code>horacio@my-laptop:~$ ssh ubuntu@51.68.xxx.xxx
-Enter passphrase for key '/home/horacio/.ssh/id_rsa':
-Welcome to Ubuntu 21.04 (GNU/Linux 5.11.0-18-generic x86_64)
+![Setting-up PCI - SSH connection information](images/vrack-example-05-bis.png){.thumbnail}
 
-  System information as of Fri Jun 18 04:37:54 UTC 2021
+If we log in to the instance using SSH, we can see that it has two network interfaces, one attached to the public IP address we use to log in, the other attached to the private network:
```
Comment (OVH-Team-Guides) [inline pages/platform/kubernetes-k8s/vrack-example-k8s-and-pci/guide.en-gb.md]: "the public IP address we use to log in" -> "the public IP address we used to log in"

File: pages/platform/kubernetes-k8s/backing-up-volumes-using-stash/guide.en-asia.md (hunk 1 above)
Comment (scraly) [reply pages/platform/kubernetes-k8s/backing-up-volumes-using-stash/guide.en-asia.md]: sorry, I fixed it :)

File: pages/platform/kubernetes-k8s/installing-helm/guide.en-asia.md (hunk 2 above)
Comment (scraly) [reply pages/platform/kubernetes-k8s/installing-helm/guide.en-asia.md]: fixed

File: pages/platform/kubernetes-k8s/vrack-example-k8s-and-pci/guide.en-gb.md (hunk 3 above)
Comment (scraly) [reply pages/platform/kubernetes-k8s/vrack-example-k8s-and-pci/guide.en-gb.md]: fixed

File: pages/platform/kubernetes-k8s/vrack-example-k8s-and-pci/guide.en-gb.md (hunk 4 above)
Comment (scraly) [reply pages/platform/kubernetes-k8s/vrack-example-k8s-and-pci/guide.en-gb.md]: fixed

**Comment by scraly** [comment]
Fixed all your remarks, thanks :)

**Comment by OVH-Team-Guides** [comment]
Thanks for your contribution and the fixes @scraly

**Pull request merged**